lib/twurl/app_only_oauth_client.rb

```diff
@@ -57,12 +57,9 @@ def set_http_client_options(http)
       http.set_debug_output(Twurl.options.debug_output_io) if Twurl.options.trace
       http.read_timeout = http.open_timeout = Twurl.options.timeout || 60
       http.open_timeout = Twurl.options.connection_timeout if Twurl.options.connection_timeout
-      # Only override if Net::HTTP support max_retries (since Ruby >= 2.5)
-      http.max_retries = 0 if http.respond_to?(:max_retries=)
-      if Twurl.options.ssl?
```

Comment:
was removing the SSL option to always turn off SSL verification intentional?

Reply:
I believe so - the API requires HTTPS / TLS connections.